.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,64 @@
+name: Continuous Integration
+on: [push, pull_request]
+
+jobs:
+  rustfmt:
+    name: Formatting check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Rust stable
+      uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
+          profile: minimal
+          components: rustfmt
+    - name: Run cargo fmt
+      uses: actions-rs/cargo@v1
+      with:
+        command: fmt
+        args: --all -- --check
+
+  clippy:
+    name: Analyzing code with Clippy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Rust stable
+      uses: actions-rs/toolchain@v1
+      with:
+          toolchain: stable
+          profile: minimal
+          components: clippy
+    - name: Run cargo clippy
+      uses: actions-rs/cargo@v1
+      with:
+        command: clippy
+        args: --workspace --all-features -- -D warnings
```

Comment:
Don't you want to add tests? (maybe all-targets) ?

Reply:
Done with `--all-targets` in 669c36b0c49339b4676579bde32db0ed4c1672ba.